csrankings-p.csv

```diff
@@ -353,8 +353,8 @@ Paul R. Tennent,University of Nottingham,https://www.nottingham.ac.uk/computersc
 Paul Ralph,Dalhousie University,http://paulralph.name,oRBuFa0AAAAJ
 Paul Rayson,Lancaster University,https://www.lancaster.ac.uk/scc/about-us/people/paul-rayson,DACFB-kAAAAJ
 Paul Richmond,University of Sheffield,https://www.sheffield.ac.uk/dcs/people/academic/paul-richmond,fLS64xsAAAAJ
-Paul Rosen,University of South Florida,http://www.cspaul.com/wordpress,6iv6AEcAAAAJ
-Paul Rosen 0001,University of South Florida,http://www.cspaul.com/wordpress,6iv6AEcAAAAJ
+Paul Rosen,University of Utah,http://www.cspaul.com/wordpress,6iv6AEcAAAAJ

```

Comment:
This entry is out of date, please remove (see DBLP).

Reply:
Fixed.

Reply:
Also update Paul A. Rosen 0001.

Reply:
Done.